benchmark/implementations/markdown-it-2.2.1-commonmark/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var md = require('../../extra/lib/node_modules/markdown-it')('commonmark');
+var md = require('markdown-it')('commonmark');
```

Comment:
As far as i see, this benchmark is really for 2.2.1 version, so this change is invalid.

I'd suggest to close PR, because `make benchmark-deps` should help.